src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/SendPostNotificationHandlerTest.java

```diff
@@ -78,7 +78,8 @@ void it_can_handle_callback() {
 
```

Comment:
Test should_notify_case_officer_that_case_is_listed()
should include for event REMOVE_LEGAL_REPRESENTATIVE, REMOVE_REPRESENTATION
Suggest using @ParameterizedTest for event

Reply:
Good spot, included for the other events